ci-operator/step-registry/openshift-qe/installer/gcp/ipi/sdn/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
think this Pr didn't tag anyone because these owners files arent set up properly. Each "link" should be pointing back to the OWNERS file under the base ["openshift-qe"](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/OWNERS) folder

Think you'll need multiple "../" for each sub folder you added
Example: https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/installer/aws/ovn/OWNERS